lfucg/lexingtonky.gov
# Update styles on news body width, cards on different screens

## Summary / Approach

Updated the news main body content to be centered with a width of 940px + 10% of screen to account for the padding. 
Added some UX to the cards in a scrollable horizontal div to help with screen height and the shadows should help with UX and indication there is a scrollable div. 

## Testing Instruction(s)

- Check out homepage on mobile
- Go to news article on mobile and desktop to see changes. 

## Screenshots

<img width="1728" alt="Screenshot 2023-06-30 at 10 31 01 AM" src="https://github.com/lfucg/lexingtonky.gov/assets/27921423/f49f9421-167c-4da8-90c6-ba821074e9b9">
<img width="354" alt="Screenshot 2023-06-30 at 10 31 13 AM" src="https://github.com/lfucg/lexingtonky.gov/assets/27921423/dd5e2bdd-3f8f-4041-8119-aec82cf35d24">

## Metadata

| Question | Answer |
|----------|--------|
| Did you perform a self-review of this PR? | yep
| Documentation reflects changes? | n/a
| Unit/Functional tests reflect changes? |n/a
| Did you perform browser testing? | yeah, chrome FF
| Did you provide detail in the summary on how and where to test this branch? | yep
| Relevant links | https://apaxsoftware.atlassian.net/browse/LG-106
